main.go

```diff
@@ -111,17 +112,27 @@ func localSetup(localPath string, config sequinsConfig) *sequins {
 }
 
 func s3Setup(bucketName string, path string, config sequinsConfig) *sequins {
+	metadata := ec2metadata.New(session.New())
```

Comment:
Ha, I had just typed a comment suggesting this but github didn't let it go through. then on refresh, this was here. +1 on great minds thinking alike (...this time at least)